ansible/roles/was_automation_register/defaults/main.yml

```diff
@@ -3,26 +3,25 @@ api_key: 'thisIsAlwaysOverridden'
 wa_cert: |
          -----BEGIN CERTIFICATE-----
-         MIIDozCCAougAwIBAgIRAMUFiOopB4Qq6TC7jjyMmxgwDQYJKoZIhvcNAQELBQAw
-         PjEVMBMGA1UEChMMY2VydC1tYW5hZ2VyMSUwIwYDVQQDExxJQk0gQXV0b21hdGlv
-         biBGb3VuZGF0aW9uIENBMB4XDTIxMDYwMTIyNTQ1NVoXDTIxMDgzMDIyNTQ1NVow
-         JTEVMBMGA1UEChMMY2VydC1tYW5hZ2VyMQwwCgYDVQQDEwNjcGQwggEiMA0GCSqG
-         SIb3DQEBAQUAA4IBDwAwggEKAoIBAQCv4Zk6hknNRwhhTcFqyuZ9Xv8iIjkpOslH
-         JGxZVZJ7egg4JhHRyuvWDfh9DcPFDxnb66UuP/4zqLlV92YpNSwH5Z/9nK0Ijv6g
-         CAGhuM59aDYIP2b6ZT8QKFY5Pxq2UiHAfKllSQTMDi7EJ1W/mzn4xiNtbOiCHYtv
-         6Y1+nZfbl1b3r3VjF8TIESb51R4gp0b3JQvWY9Ww314tIaw2Uu3IRmSzz77SDhSn
-         RvkMraVdrKAmbULx2vWzf4uSmwdV6IHbuoBCr+rR1bGl1diBQrAtp7KezwlOg/z3
-         8EUn9Ex83kygRiDLsU2VY+ER4mtx2LgJgsVhVRk2+f8YXZzVj/K/AgMBAAGjgbQw
-         gbEwDgYDVR0PAQH/BAQDAgWgMBMGA1UdJQQMMAoGCCsGAQUFBwMBMAwGA1UdEwEB
-         /wQCMAAwfAYDVR0RBHUwc4IDY3BkghhjcGQud2Vic3BoZXJlLWF1dG9tYXRpb26C
-         HGNwZC53ZWJzcGhlcmUtYXV0b21hdGlvbi5zdmOCNGNwZC13ZWJzcGhlcmUtYXV0
-         b21hdGlvbi5hcHBzLnBlcm9uaS5jcC5meXJlLmlibS5jb20wDQYJKoZIhvcNAQEL
-         BQADggEBAD5CoQdhPsozojHmkML2Tv7gF/qxOdTBdjfZlxKSr3Yc6a3vLVTdvkN6
-         XPgfdru+v8I5SYxHnWWONTidckFKnKYMJLR6jJvVuBQlkEWWJhA6LIJFXzNTlDo4
-         S8OiK7erum3J2pZWh6rQZ5bcecIHP3M3zrXxQ72zugy36an782A7bUw8qsfQdB5f
-         uoQ+Leq1vtmoBPL2cM6i3b6Y63+BMvw57z7eJo11r5TuAkpeu7uNPQq+tHCPctVk
-         nuH+bEAhdzgVVRj4RDcuGcHjzI0XGHQrSwRWoMvY7IEmT7UqFZRp8a63jiDT6Qtw
-         Pr1gsnssiL4XAXqSvN7jhkduU+JZ3hI=
+         MIIDYDCCAkigAwIBAgIRANYE8LObslftG7BUoz4zvC4wDQYJKoZIhvcNAQELBQAw
+         JzElMCMGA1UEAxMcSUJNIEF1dG9tYXRpb24gRm91bmRhdGlvbiBDQTAeFw0yMjA1
+         MjcxODUyMzhaFw0yMjA4MjUxODUyMzhaMA4xDDAKBgNVBAMTA2NwZDCCASIwDQYJ
+         KoZIhvcNAQEBBQADggEPADCCAQoCggEBANyu9n4eZg+Ef7amYzonaENDyF5bgCkL
+         wrw1QZtCC4vf8aothRzwyWOvWCo3HfHb2Nk+ITCOqH4JwlTs+3cVJyfavvMj87+h
+         begdYyy2nl6JdqV6MUMuaGwBuB3+NwoO1mat9tg0MEj0E7xoiVSg0PtN1wECuZm3
+         LYreh/nEdX6wWImC2q/jTtwSVR9cRcZ03BXWI3qZmbgHl/C/PwgtbrOM5dRpqaXk
+         a1e9RDIqs48RuArHemHDryEiWBIVJ43uW91MxT2dwYwTfmiGNTKj3BZGkrd+RrIb
+         6P9qCzJxbbhLo4S/js9K8mn1iqPZi+HqizKFnikX5T4DftO5ZlAi2hECAwEAAaOB
+         nzCBnDAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDAYDVR0T
+         AQH/BAIwADAfBgNVHSMEGDAWgBRgKetUoPJYtq9U0uvXdrgVwqOLszBGBgNVHREE
+         PzA9ggNjcGSCGGNwZC53ZWJzcGhlcmUtYXV0b21hdGlvboIcY3BkLndlYnNwaGVy
+         ZS1hdXRvbWF0aW9uLnN2YzANBgkqhkiG9w0BAQsFAAOCAQEAx5F8MOeSQHv0AegH
+         lEFFVygMuryK9yH9G8DLWZCQAysba/k5HtqYmpEFpEnfAPXrriMm3o96dSFzhOzx
+         lsxS78aU1QBRyG2IdiOVVNJ9ZsZL6xa92Xu5qp/5kMIpZ5Rp9J8Z3itZiFhozdKM
+         6b1gLDzDAb3vTzENxK33kjccXF09NgS6jqMsy2YYVsM9YfWiZ/PfeFg46OKS/YBZ
+         N11EIHGYMESs1NIP9J5JgeunxgKrV3UYeXahyw3SqXGmxFnmxVnoGch/OFSKWRkf
+         416ZDnB5YQPksujrvKOKFpp8GGidIZHOv9QmovkflkkBzDLxoBYJxM/ikT/r3lrJ
+         v7gTWQ==
          -----END CERTIFICATE-----
 wa_keystore_pass: 'letsHideSomePrivateStuff'
```

Comment:
Even though this is likely overidden I think I would rather fail than let it use this default